views/index.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <center>
```

Comment:
`<center>` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center). see if you can find a way to do the same thing using CSS